streams/src/test/java/org/apache/kafka/streams/processor/internals/StoreChangelogReaderTest.java

```diff
@@ -416,6 +417,7 @@ public void shouldPollWithRightTimeoutWithStateUpdater(final Task.TaskType type)
         shouldPollWithRightTimeout(true, type);
     }
 
+    @Disabled
```

Comment:
I would not disable this test, but rather clean it up directly. Or exclude `StoreChangelogReader` from this PR.